src/app/pages/PhotoGalleryPage/PhotoGalleryPage.jsx

```diff
@@ -38,6 +37,7 @@ import ATIAnalytics from '../../components/ATIAnalytics';
 import ChartbeatAnalytics from '../../components/ChartbeatAnalytics';
 import LinkedData from '../../components/LinkedData';
 import { ServiceContext } from '../../contexts/ServiceContext';
+import disclaimer from '../../components/Disclaimer';
```

Comment:
**nitpick** (as I realise it was already like this): I thought component were supposed to be in capital case?